bin/upgrade/dev/upgrade_fs.sh

```diff
@@ -5,13 +5,29 @@ echo "Running filesystem modifications"
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../.." && pwd )"
 CUR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
+#upgrade nodejs
+if [ -f /etc/redhat-release ]; then
+	curl -sL https://rpm.nodesource.com/setup_10.x | bash -
```

Comment:
Awesome :) I guess if we're officially Node 10 now, the Dockerfiles will need an update too